src/UI/templates/default/Divider/divider.less

```diff
@@ -0,0 +1,12 @@
+h6.il-divider {
+	padding: @padding-small-vertical @padding-small-horizontal;
+	background-color: darken(@il-secondary-container-bg, 5%);
```

Comment:
Do not use lighten in any other files than variables.less. See: https://github.com/ILIAS-eLearning/ILIAS/tree/trunk/templates#colors

Introduce an il-divider-bg color for this.